lib/kovid/helpers.rb

```diff
@@ -14,4 +16,21 @@ def dateman(date)
     date_to_parse = Date.strptime(date, '%m/%d/%y').to_s
     Date.parse(date_to_parse).strftime('%d %b, %y')
   end
+
+  # Insert commas to format numbers
```

Comment:
I think this is a good intention to describe methods. However, some methods are pretty self-explanatory, making the comments redundant. You can discern what this is doing just be quickly looking at the code right? What do you think?

Reply:
I thought adding some basic doc would be helpful in this case because inside this method there is a chain of lots of other methods being called.  
But I guess, like you said, it is not strictly necessary, because the sequence is not so complicated.